tests/client/test_file_chunk.py

```diff
@@ -0,0 +1,7 @@
+from qingstor.sdk.client.file_chunk import FileChunk
+part_size=20
+with open('test_fileS.txt','rb') as f:
+    output=FileChunk(part_size,f)
+    for part_index in range(output.part_amount):
+        cur_part_content=output.read_file_part(part_index)
+        print cur_part_content
```

Comment:
What's that for ?
Read this <https://docs.python.org/3/library/unittest.html> to know more about unit testing.